app/models/concerns/user/onboarding.rb

```diff
@@ -12,16 +12,12 @@ def onboarding_required_fields
   end
 
   def onboarding_completed?
-    # cache the result so we don't have to check all the fields on every request
-    Rails.cache.fetch("user/#{hashid}/onboarding_completed?", expires_in: 12.hours) do
-      onboarding_required_fields.all? { |field| send(field).present? }
-    end
+    onboarding_required_fields.all? { |field| send(field).present? }
```

Comment:
caching wont work cause when they just completed the onboarding it'll look for cached result which will be wrong